dali/pipeline/operators/decoder/nvjpeg_helper.h

```diff
@@ -51,6 +51,19 @@ namespace dali {
   } while (0)
 
 struct StateNvJPEG {
+  StateNvJPEG(nvjpegHandle_t handle, nvjpegJpegDecoder_t decoder_host, nvjpegJpegDecoder_t decoder_hybrid) {
+    NVJPEG_CALL(nvjpegBufferPinnedCreate(handle, nullptr, &pinned_buffer));
+    NVJPEG_CALL(nvjpegDecoderStateCreate(handle, decoder_host, &decoder_host_state));
+    NVJPEG_CALL(nvjpegDecoderStateCreate(handle, decoder_hybrid, &decoder_hybrid_state));
+    NVJPEG_CALL(nvjpegJpegStreamCreate(handle, &jpeg_stream));
+  }
+
+  ~StateNvJPEG() {
+    nvjpegJpegStreamDestroy(jpeg_stream);
+    nvjpegJpegStateDestroy(decoder_hybrid_state);
```

Comment:
These functions return error code, which should be interpreted. Please provide implementation

Reply:
Right, we should std::terminate if anything went wrong here